src/WorkOS.net/Services/_common/Entities/WorkOSList.cs

```diff
@@ -19,7 +19,7 @@ public class WorkOSList<T>
         /// <summary>
         /// Pagination cursor options.
         /// </summary>
-        [JsonProperty("listMetadata")]
-        public ListMetadata ListMetadata { get; set; }
+        [JsonProperty("List_Metadata")]
+        public List_Metadata List_Metadata { get; set; }
```

Comment:
We should keep the field name the same.

```suggestion
        public ListMetadata ListMetadata { get; set; }
```